template.yaml

```diff
@@ -0,0 +1,67 @@
+Description: Upload an object to an S3 bucket, triggering a Lambda event, returning the object key as a Stack Output.
+Parameters:
+  Key:
+    Description: S3 Object key
+    Type: String
+    Default: test
+  Body:
+    Description: S3 Object body content
+    Type: String
+    Default: TEST CONTENT
+  abc1:

```

Comment:
Please use names which are easy to understand and self explanatory. You can use BucketName here.